covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -717,20 +718,11 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
                 <p className={styles["action-task-progress-code"]}>
                   {props.spec.prolificCompletionCode}
                 </p>
-                <Button
-                  type="primary"
-                  icon={<ExportOutlined />}
-                  className={styles["action-task-progress-completion-button"]}
-                  onClick={(event: MouseEvent) => {
-                    window.open(
-                      "https://app.prolific.com/submissions/complete?cc=" +
-                        props.spec.prolificCompletionCode,
-                      "_blank"
-                    )
-                  }}
-                >
-                  Take me to Prolific Academic
-                </Button>
+                {props.renderSubmitButton({
```

Comment:
Can you explain this change here?

Reply:
Sure! It turns out that, within the 100s of level of abstraction, there is this function which returns a `<Button>` React component whose onClick function is already linked to functions which take care of emitting to the server, through the API, the change of node status into FINISHED (among other things). So, instead of creating my own button, I reused that function to slightly reduce the code and for future maintenance purposes.